src/services/member/plugins/action/test/service.test.ts

```diff
@@ -39,22 +40,32 @@ describe('Action member service', () => {
     it('throw for signed out user', async () => {
       ({ app } = await build({ member: null }));
 
-      await service.getFilteredActions(actor, buildRepositories(), {}).catch((e) => {
-        expect(e).toBeInstanceOf(UnauthorizedMember);
-      });
+      await getActionMemberService()
+        .getFilteredActions(actor, buildRepositories(), {})
+        .catch((e) => {
+          expect(e).toBeInstanceOf(UnauthorizedMember);
```

Comment:
What we're doing here is a bit confusing to me. I know you're inheriting what was implemented before
If the function doesn't throw an exception, then the test will pass (That's how I read it)

What about :


```suggestion
      await expect(
        getActionMemberService().getFilteredActions(actor, buildRepositories(), {}),
      ).rejects.toBeInstanceOf(UnauthorizedMember);
```

Reply:
Yes it is more understandable 🤩